src/modules/mail/templates/booking-status.template.ts

```diff
@@ -36,6 +36,10 @@ export const generateBookingStatusMail = (booking: Booking) => {
         <td style="padding: 8px; border: 1px solid #ddd; background-color: #f1f1f1; font-weight: bold;">Check-out</td>
         <td style="padding: 8px; border: 1px solid #ddd;">${checkOutDateStr}</td>
       </tr>
+      <tr>
+        <td style="padding: 8px; border: 1px solid #ddd; background-color: #f1f1f1; font-weight: bold;">Service fee</td>
+        <td style="padding: 8px; border: 1px solid #ddd;">${totalPrice / 10} USD</td>
```

Comment:
can you format it. it should show only 2 decimal places. you can take a look at how I did it in the bookings service.